interfaces/seccomp/backend.go

```diff
@@ -81,6 +83,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err != nil {
 		return fmt.Errorf("cannot synchronize security files for snap %q: %s", snapName, err)
 	}
+
+	for baseName := range content {
+		in := filepath.Join(dirs.SnapSeccompDir, baseName)
+		out := filepath.Join(dirs.SnapSeccompDir, baseName+".bpf")
```

Comment:
I like how `/var/lib/snapd/seccomp` is preserved in @niemeyer's suggestion to use `/var/lib/snapd/seccomp/bpf/...` since finding the subdir `bpf/` is more discoverable than the sibling dir `seccomp.bpf`.

As mentioned in previous days, I don't have a strong preference on the naming approach, so what is proposed is fine with me, but I got to wondering if we have the `bpf/` subdir and both the input and output files are there, we don't really need *both* the input and ouput files to have a suffix. Eg, could simply use _seccomp/bpf/snap.name.cmd.src_ (or _seccomp/bpf/snap.name.cmd.in_) and _seccomp/bpf/snap.name.cmd_ (ie, no extension for the output file). Or flip it and have no extension for the input file and use _.cache_ or _.bin_ for the output. I suspect that the former (have an extension on the input file) is preferred since it retains the hint @niemeyer was after.

Not blocking on this if you're happy with *.src and *.bin, just thought it was potentially cleaner to use an extension on only one of input or output.